state/logs.go

```diff
@@ -962,20 +965,13 @@ func getCollectionCappedInfo(coll *mgo.Collection) (bool, int, error) {
 	if err != nil {
 		return false, 0, errors.Trace(err)
 	}
-	// For mongo 2.4, the "capped" key is only available when it is set
-	// to true. For mongo 3.2 and 3.6, it is always there.
-	// If capped is there, but isn't a bool, we treat this as false.
-	capped, _ := stats["capped"].(bool)
+	capped := stats["capped"].(bool)
```

Comment:
doing this without the , ok will panic if it doesn't cast correctly.
I'd rather at least keep that check in place.